pkg/idp/provider/github/manifest/flow.go

```diff
@@ -128,6 +128,8 @@ func (f *Flow) run() error {
 			cancel()
 		})
 
+		mux.Handle("/static/", http.FileServer(http.FS(f.renderer.fs)))
```

Comment:
Asset files (i.e. `main.css`) weren't being served, so I added this to try and serve the `static` directory. It works for the initial page, however on redirect back to the `/complete` page, the context is cancelled before the browser's request for assets can be fulfilled.

@anvddriesch I figured you'd know a better way to handle this 😄